src/CoreBundle/private/library/classes/TCMSFields/TCMSField.class.php

```diff
@@ -759,6 +760,7 @@ public function _GetSQLDefinition(&$postData = null)
         } else {
```

Comment:
What happens if $lengthSet remains empty?

Reply:
That is ok. Not every type has a length. That was the problem in the first place: it was set for a type, where it should have been empty.